end-to-end-test/remote/specs/core/oncoprint.screenshot.spec.js

```diff
@@ -198,8 +198,11 @@ describe('track group headers', function() {
     it('oncoprint should delete clustered heatmap group correctly', function() {
         // Remove the mrna heatmap group, leaving the methylation group and everything sorted by data
         var mrnaElements = getGroupHeaderOptionsElements(2);
-        browser.click(mrnaElements.button_selector);
-        browser.waitForVisible(mrnaElements.dropdown_selector, 1000); // wait for menu to appear
+        setDropdownOpen(
+            true,
+            mrnaElements.button_selector,
+            mrnaElements.dropdown_selector
+        );
```

Comment:
looks like this one still fails? https://circleci.com/gh/cBioPortal/cbioportal-frontend/62692?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link